codecov_cli/commands/labelanalysis.py

```diff
@@ -285,20 +285,20 @@ def _send_labelanalysis_request(payload, url, token_header):
     return eid
 
 
-def _dry_run_output(result: LabelAnalysisRequestResult):
-    logger.info(
-        "Not executing tests because '--dry-run' is on. List of labels selected for running below."
-    )
-    logger.info("")
-    logger.info("Label groups:")
-    logger.info(
-        "- absent_labels: Set of new labels found in HEAD that are not present in BASE"
+def _dry_run_output(
+    result: LabelAnalysisRequestResult, runner: LabelAnalysisRunnerInterface
+):
+    labels_to_run = list(
+        set(
+            result.absent_labels
+            + result.global_level_labels
+            + result.present_diff_labels
+        )
     )
-    logger.info("- present_diff_labels: Set of labels affected by the git diff")
-    logger.info("- global_level_labels: Set of labels that possibly touch global code")
-    logger.info("- present_report_labels: Set of labels previously uploaded")
-    logger.info("")
-    logger.info(json.dumps(result))
+    output = runner.dry_run_runner_options + sorted(labels_to_run)
```

Comment:
is the `--cov-context=test` necessary?

Reply:
it is *absolutely* necessary that users run pytest with this. Other tools will probably have their own quirks.

I don't want to take the chance of not including these options and just tell users to add them on their own. It's too much of a chance that someone will forget to do it and then ATS won't work.

Reply:
`--cov-context=test` is what tells coveragepy to track which tests cover which line?

Reply:
Yes, that is correct.